### PR TITLE
add kepstitch tutorial

### PR DESCRIPTION
**Pull request by mirca**
https://keplerscience.arc.nasa.gov/PyKEprimerWalkthroughD.shtml

### CODE REVIEW

**Comment by barentsen** [comment]
Let's make sure to have a look at Section 6 in Kinemuchi et al 2012 (http://adsabs.harvard.edu/abs/2012PASP..124..963K), which offers a clear description of three difference approaches to stitch quarters together.  We may want to include and compare all three approaches as part of this tutorial, and borrow language from the paper.

**Comment by mirca** [comment]
@barentsen I guess we can merge this and make additions latter if necessary?

**Comment by barentsen** [comment]
I'm ok with that!

File: docs/source/tutorials/removing-stellar-astroph-quarter-stitch.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+..
+
+Example 4: Removing Stellar Astrophysics and Quarter Stitching
+==============================================================
+
+The purpose of this tutorial is to flatten both systamtic artifacts and
```
Comment (barentsen) [inline docs/source/tutorials/removing-stellar-astroph-quarter-stitch.rst]: typo: systamtic => systematic

**Pull request merged**